core/pubnub_internal_common.h

```diff
@@ -3,6 +3,7 @@
 #if !defined INC_PUBNUB_INTERNAL_COMMON
 #define INC_PUBNUB_INTERNAL_COMMON
 
+#include <stdbool.h>
```

Comment:
I see many random import changes. 
Probably they're related to the architecture and feature sets. Am I right?

Reply:
Some imports have been removed because IDE shown that they are unused, some added in other place.

To be honest, `<stdbool.h>` and `core/c99/stdbool.h`: not sure about intention. Do we still need the latest c89 support (as I understand this is the only reason why `core/c99/stdbool.h` exists because `bool` is missing from C89 std)?

Moreover, additionally I was confused by feature flags which let specify custom booleans at a compile-time and wasn't sure whether it needed to be utilized in any way.

Reply:
>  Do we still need the latest c89 support 
This topic is still under the discussion. :/